src/__swaps__/screens/Swap/resources/assets/userAssets.ts

```diff
@@ -108,15 +109,18 @@ async function userAssetsQueryFunction({
 
     return parsedAssetsDict;
   }
+
   const cache = queryClient.getQueryCache();
   const cachedUserAssets = (cache.find(userAssetsQueryKey({ address, currency, testnetMode }))?.state?.data ||
     {}) as ParsedAssetsDictByChain;
   try {
-    const url = `/${SUPPORTED_CHAIN_IDS({ testnetMode }).join(',')}/${address}/assets`;
+    staleBalancesStore.getState().clearExpiredData(address);
+    const staleBalanceParam = staleBalancesStore.getState().getStaleBalancesQueryParam(address);
+    let url = `/${SUPPORTED_CHAIN_IDS({ testnetMode }).join(',')}/${address}/assets?currency=${currency.toLowerCase()}`;
+    if (staleBalanceParam) {
+      url += url + staleBalanceParam;
```

Comment:
`url += url + staleBalanceParam;` shouldn't be a `+=`

this was a bug in the original code that has since been fixed in develop